src/org/opendatakit/briefcase/util/PushFromServerException.java

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This file should have gone with the first commit... Sorry!

Reply:
If you want, it should be pretty easy to fix. I would do an interactive rebase, leave this commit open for editing, `git reset PushFromServerException.java` to unstage this one, `git stash`, `git rebase --continue` then interactive rebase again leaving the first commit open, `git stash apply`, `git add -A`, git commit --amend`. Ok, maybe it's not super easy...

Reply:
I think you're right! I'll do that :)

Reply:
OK, I did it but I had to do something different:
- Interactive rebase of https://github.com/opendatakit/briefcase/pull/414/commits/77dba135eb2f8f2742682b35b0fd1cb38aca64bb to edit it
- Since git first applies the commit and then it lets you edit, I had to `git rm` the file
- Interactive rebase of https://github.com/opendatakit/briefcase/pull/414/commits/42b7a5c7e1399abb58e4dd6e3c258051814484b1 to edit it
- Create the file from the raw version in https://github.com/opendatakit/briefcase/pull/414/commits/77dba135eb2f8f2742682b35b0fd1cb38aca64bb that still exists on GitHub
- Profit :1st_place_medal:

Reply:
Oh, I'm surprised the `reset` didn't do the trick! But hey, you got to the most important step. 🥇 🤣

Reply:
It turns out that I could have done a `git reset HEAD~` or maybe even a `git reset PushFromServerException.java HEAD~`